common/src/thrift/tachyon.thrift

```diff
@@ -147,15 +108,15 @@ service BlockMasterService {
 
   void workerRegister(1: i64 workerId, 2: list<i64> totalBytesOnTiers,
       3: list<i64> usedBytesOnTiers, 4: map<i64, list<i64>> currentBlocksOnTiers)
-    throws (1: TachyonException te)
+    throws (1: TachyonTException e)
```

Comment:
I think as long as the outermost client methods can throw an IOException and a TachyonException, which they all can, we won't have to change any user-facing APIs. And I'd prefer to keep the internal APIs as strict about exceptions as possible, since it helps check correctness and refactoring them shouldn't be too difficult for us. As far as we know, a new RPC implementation might start throwing some totally different exceptions (maybe different than an IOException) anyways, which we'd have to modify our code to handle, so I don't think we can make migrating the RPC layer perfectly seamless.